packages/grafana-data/src/utils/url.ts

```diff
@@ -156,7 +155,7 @@ export function parseKeyValue(keyValue: string) {
 
         let parsedVal: any;
         if (typeof val === 'string' && val !== '') {
-          parsedVal = val === 'true' || val === 'false' ? val === 'true' : toNumber(val);
+          parsedVal = val === 'true' || val === 'false' ? val === 'true' : val;
```

Comment:
I think `number` should be removed from [UrlQueryValue](https://github.com/grafana/grafana/blob/b603a8dbb784c43a83e0316d9501cc8191add760/packages/grafana-data/src/utils/url.ts#L12) type, but then some type checks are failing, so I guess some places may require extra casting to numbers (?)

Reply:
Hmm, that is a public api. Not sure we want to change that in such a short notice.

Reply:
Or maybe I misunderstood it. Is `UrlQueryValue` used as input (from URL) and output type (to update URL)?